scripts/postlink/ios/postlink.js

```diff
@@ -4,8 +4,20 @@ var inquirer = require('inquirer');
 var path = require("path");
 var plist = require("plist");
 
+// Fix for https://github.com/Microsoft/react-native-code-push/issues/477
+// Try to resolve package.json, assuming we are working off of the project root
+// We will use this later to fetch the correct AppDelegate and Plist file, in case there are multiple.
+var package = require(path.resolve('./package.json'));
```

Comment:
Correct me if I'm wrong, but I think the end result would be the same if this was just `require("./package.json");` (note the double quotes also).

Also, I don't think you need the comment above - the comment below does a great job explaining everything.